src/components/cards-grid-list/docs.mdx

```diff
@@ -17,18 +17,18 @@ The layout for this component is driven by minimum width settings on card childr
 
 <LiveComponent>{`
 <CardsGridList isOrdered={false}>
-  <div style={{ border: '1px solid magenta', padding: '16px' }}>
+  <li style={{ border: '1px solid magenta', padding: '16px' }}>
```

Comment:
Have updated Swingset to make it more clear that children of `CardsGridList` should be `<li />`.